### DISCUSSION TITLE
My github copilot not working, failed to connect server

### DISCUSSION THREAD

**Discussion by devvsakib** [community/community#43761] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

A few days ago I installed the GitHub copilot extension CZ I have a GitHub student account. But today GitHub copilot not working and it shows I have no access to GitHub copilot. Is there any way to solve this or just expired my trial or something?
```
[INFO] [auth] [2023-01-09T18:57:14.897Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-01-09T18:57:14.900Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

```

**Comment by iamcjwilliams** [comment, marked as answer]
@devvsakib Please refer to this [post](https://github.com/community/community/discussions/43987)
